portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,42 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
```

Comment:
I see you've implemented this, but it's not used in index.html. Just wanted to check if that was intentional?

Reply:
I actually edited this but I'm planning to delete it.